WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -188,6 +188,17 @@ public object GetStatus()
 		};
 	}
 
+	[JsonRpcMethod("getcoinjoinstatus")]
+	public CoinJoinClientState GetCoinjoinStatus()
+	{
+		var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+
+		AssertWalletIsLoaded();
+		var coinJoinManager = Global.HostedServices.Get<CoinJoinManager>();
+		_ = coinJoinManager.TryGetWalletStatus(activeWallet.WalletName, out var walletCoinjoinStatus);
+		return walletCoinjoinStatus ?? CoinJoinClientState.Idle;
```

Comment:
Sorry for giving you feedback in tiny drops but it is just that now i see it more clear. Do not returns ints for enumerations because those are internal values.  Instead, do something like this:

```
			torStatus = sync.TorStatus switch
			{
				TorStatus.NotRunning => "Not running",
				TorStatus.Running => "Running",
				_ => "Turned off"
			},
```

Reply:
Don't worry. I made 19c819f.
In the case of `null`, it returns `Not managed`